lang/csharp/src/apache/main/Generic/GenericRecord.cs

```diff
@@ -239,7 +239,7 @@ public override string ToString()
                 sb.Append(contents[field.Pos]);
                 sb.Append(", ");
             }
-            sb.Append("}");
+            sb.Append('}');
```

Comment:
The latest msbuild 16.8 has many new warnings (which are mostly seful/valid). The compiler was complaining that StringBuilder.Append(char) should be used if the string arg is a single character. It is more efficient I guess, sinve only a char has to be passed on the stack to the function. There were many Append(string with 1 char) calls, and all of them caused a warning-> error since all warnings are treated as errors.

Reply:
```
error CA1834: Use 'StringBuilder.Append(char)' instead of 'StringBuilder.Append(string)' when the input is a constant unit string
```

Reply:
Ah, that makes sense. Thanks 👍